ansible/roles/schulcloud-server-core/tasks/main.yml

```
@@ -186,6 +186,13 @@
       template: admin-api-server-svc.yml.j2
     when: WITH_API_ADMIN is defined and WITH_API_ADMIN|bool
 
+  - name: TlDraw server Secret (from 1Password)
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: tldraw-server-onepassword.yml.j2
+    when: ONEPASSWORD_OPERATOR is defined and ONEPASSWORD_OPERATOR|bool
```

We want only the TlDraw 1Password Item Deployed if TlDraw is activate for an Instance
```suggestion
    when: 
     - ONEPASSWORD_OPERATOR is defined and ONEPASSWORD_OPERATOR|bool
     - WITH_TLDRAW is defined and WITH_TLDRAW|bool
```